inc/views/base/error/no_permission.twig

```diff
@@ -0,0 +1,36 @@
+{{ lang.error_nopermission_guest_1 }}
+<ol>
+    <li>{{ lang.error_nopermission_guest_2 }}</li>
+    <li>{{ lang.error_nopermission_guest_3 }}</li>
+    <li>{{ lang.error_nopermission_guest_4 }}{% if mybb.user.uid %} (<a href="member.php?action=resendactivation">{{ lang.error_nopermission_user_resendactivation }}</a>){% endif %}</li>
+    <li>{{ lang.error_nopermission_guest_5 }}</li>
+</ol>
+
+{% if mybb.user.uid > 0 %}
+    <br />
+    {{ trans('error_nopermission_user_username', mybb.user.username) }}
+{% else %}
+
+    <form action="member.php" method="post">
+        <input type="hidden" name="action" value="do_login" />
+        <input type="hidden" name="url" value="{{ redirect_url }}" />
```

Comment:
I'm not 100% sure, but we should maybe be using `|e('url')` for escaping URLs: https://twig.symfony.com/doc/2.x/filters/escape.html - we need to test it and see what it does, and if that's the case, we need to check existing translations.